services/dlab-webapp-common/src/main/java/com/epam/dlab/rest/contracts/ComputationalAPI.java

```diff
@@ -37,4 +37,5 @@
 	String COMPUTATIONAL_TERMINATE_CLOUD_SPECIFIC = COMPUTATIONAL_TERMINATE + "/cloud";
 	String COMPUTATIONAL_LIB_INSTALL = LIBRARY + COMPUTATIONAL + "/lib_install";
 	String COMPUTATIONAL_LIB_LIST = LIBRARY + COMPUTATIONAL + "/lib_list";
+	String COMPUTATIONAL_RECONFIGURE_MESSAGE = "Reconfigure compute <%s>, requested for notebook <%s>";
```

Comment:
Please rename the field to start with 'AUDIT_'.
Also, please move the field to line 24 (under the **AUDIT_MESSAGE** constant), so that all audit messages are in one place